File: src/main/Python/ticket.py
class Ticket:
    price: float
    date: datetime
    start_zone: int
    finish_zone: int
    is_luggage: bool
    place: int
    road_number: int

    def __init__(self, price, date, start_zone, finish_zone, is_luggage, place, road_number):
        self.price = price
        self.date = date
        self.start_zone = start_zone
        self.finish_zone = finish_zone
        self.is_luggage = is_luggage
        self.place = place
        self.road_number = road_number

File: src/main/java/Classes/Ticket.java
package Classes;

import java.text.DecimalFormat;
import java.util.Calendar;

public class Ticket {
    private DecimalFormat price;
    private Calendar date;
    private int startZone;
    private int finishZone;
    private boolean isLuggage;
    private int place;
    private int routeNumber;

    public Ticket(DecimalFormat price, Calendar date, int startZone, int finishZone, boolean isLuggage, int place, int routeNumber) {
        this.price = price;
        this.date = date;
        this.startZone = startZone;
        this.finishZone = finishZone;
        this.isLuggage = isLuggage;
        this.place = place;
        this.routeNumber = routeNumber;
    }
    
}
